Raspi/Test/serial_test.py

```diff
@@ -0,0 +1,50 @@
+import argparse
+import os
+import paho.mqtt.client as mqtt
+
+from test_data_generator import send_csv_data, send_fake_data
+
+# Arguments
+parser = argparse.ArgumentParser(description='DAS serial test script', add_help=True)
+parser.add_argument('-t', '--time', action='store', type=int, default=1, help="length of time to send data")
+parser.add_argument('-r', '--rate', action='store', type=float, default=0.5, help="rate of data in seconds")
+parser.add_argument('--host', action='store', type=str, default="localhost", help="address of the MQTT broker")
+parser.add_argument('-f', '--file', action='store', type=str, help="The csv file to replay. If not specified, makes up data.")
+parser.add_argument('-j', '--jump', action='store', type=int, default=0, help="Starts replaying from a specified time (in seconds)")
+
+def on_connect(client, userdata, flags, rc):
+    print("Connected to MQTT broker with result code " + str(rc))
+
+def run(args):
+    # Create pseudo-terminal pair
+    master, slave = os.openpty()
+
+    # Show ports
+    print("Master serial port:", os.ttyname(master))
```

Comment:
This is a Unix only method. https://docs.python.org/3/library/os.html#os.ttyname
We should probably check for operating system. If the user is trying to run it on Windows, let's just return a error message stating this won't work!

Reply:
Done